compaction/task_manager_module.cc

```diff
@@ -9,6 +9,8 @@
 #include "compaction/task_manager_module.hh"
 #include "replica/database.hh"
 
+namespace tasks {
```

Comment:
We usually have top-level `namespaces` correspond to the directory the source file is located in.
E.g. if some class is in namespace `sstables` it will be in the directory `sstables`.
We don't follow this as consistently as I wish but I think it is a good direction to work towards.
Having symbols in the `tasks` namespace in the `compaction` directory is confusing IMO.

Reply:
I don't even really understand why this is needed as the commit has no motivation.

Reply:
The compaction_manager::task becomes compaction::task after changes. Thus, the `tasks::` namespace is needed to differentiate between `compaction::compaction_manager_task_name` and `compaction::task_manager_task_name`. They have slightly different naming conventions, but I think it's not enough.

Reply:
Oh, I just realized that I renamed these compaction_manager::tasks anyway and put them in the both namespaces... well, that does not make sense, sorry. Will drop `namespace tasks` part.